README.md

```diff
@@ -14,15 +14,21 @@ Turn complex GraphQL queries into optimized database queries.
 pip install graphql-compiler
 ```
 
+## Quick Overview 
+
+The GraphQL compiler turns read-only queries written in GraphQL syntax to different 
+query languages. Through the GraphQL compiler, users can write powerful queries that uncover 
+deep relationships in the data while not having to worry about the underlying database query 
+language.
+
+Furthermore, the GraphQL compiler validates queries through the use of a GraphQLSchema object 
+that specifies the underlying schema of the database. We can currently autogenerate the 
+GraphQLSchema by querying an OrientDB database, (see  [Querying OrientDB with GraphQL](#querying-orientdb-with-graphql)). In the nearby future, we plan to add 
```

Comment:
Suggestion: `introspecting` instead of `querying`. Also:
- There are two spaces before the `[` of the link.
- How about putting the next sentence on its own line, to keep the file's line length low, and replacing `nearby` with `near`?